### DISCUSSION TITLE
Is there any reason why not all props are listed on the docs?

### DISCUSSION THREAD

**Discussion by moong23** [vercel/next.js#69450] [Help]
### Summary

I had a memory of using `as` props in `<Link>` component to hide(change) the URL displayed on the top URL bar.
But when I looked up at the docs, there was no `as` listed at the props, but seems like it still exists in the next.js code.
So, I want to ask if there's any reason not listing all the props at docs.

**Comment by Juneezee** [comment]
The docs is probably not the most up-to-date. However, the `as` prop is still available for the `Link` component

https://github.com/vercel/next.js/blob/ad268ef318e4f47b19e4a098404a8112ca39acb6/packages/next/src/client/link.tsx#L42-L45

**Reply by moong23** [reply]
Thx a lot :)
Docs has updated with release of app directory, but since `as` props has gone at the docs, I was wondering if it is intentional omission.

**Comment by icyJoseph** [comment]
Yeah it is a bit weird, that it is listed as a "Good to know", here, https://nextjs.org/docs/pages/api-reference/components/link

- But also, the `as` prop on link, is the same second parameter that, router push or replace take. In pages router that is, https://nextjs.org/docs/pages/api-reference/functions/use-router#routerpush

**Reply by moong23** [reply]
As you mentioned, it seems a bit odd. 
Having to thoroughly go through the entire official documentation just to find out the list of supported props is somewhat inefficient. 

Currently, I feel that if there were no props listed at the very top of the documentation, I would at least read through the additional documentation to learn about the props. However, since only the primary props are listed at the top (not really "PRIMARY" though), it was difficult to determine whether there are any additional props.
